evidence/pool.go

```diff
@@ -176,7 +192,8 @@ func (evpool *Pool) AddEvidenceFromConsensus(ev types.Evidence, time time.Time,
 		return fmt.Errorf("can't add evidence to pending list: %w", err)
 	}
 
-	evpool.evidenceList.PushBack(ev)
+	// add evidence to the consensus evidence buffer to be gossiped in the following height
+	evpool.consensusEvBuffer = append(evpool.consensusEvBuffer, ev)
```

Comment:
will we lose this evidence if power goes down?

Reply:
nope evidence is still saved in the store it just won't get pushed to the clist which is okay because on start up we recover all the evidence to the clist anyway